MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -83,6 +88,11 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getCurrentPosition(callbackId: String) {
+        let responseString = "{\"coords\":{\"latitude\":\(locationManager?.location?.coordinate.latitude ?? 0),\"longitude\":\(locationManager?.location?.coordinate.longitude ?? 0)}}"
```

Comment:
This is missing a few things from [`GeolocationPosition`](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPosition). 

Does the `CLLocationManager` provide a `timestamp`? If not, maybe we could at least generate one ourselves?

It's also missing some of the [`GeolocationCoordinates`](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationCoordinates) such as `altitude`, `heading`, `speed`, and `altitudeAccuracy`. It looks like it's fine for these properties to be null if it's not possible to retrieve them (some devices don't have these capabilities, so I think that's why they're nullable). However, there's also an `accuracy` property which can't be null. Is it possible to retrieve some value for accuracy?

Reply:
That's right. I guess I will try to add all of them.